src/annotator/test/cross-frame-test.js

```diff
@@ -29,7 +29,7 @@ describe('CrossFrame', () => {
     proxyBridge = sinon.stub().returns(fakeBridge);
 
     $imports.$mock({
-      './annotation-sync': proxyAnnotationSync,
+      './annotation-sync': { AnnotationSync: proxyAnnotationSync },
```

Comment:
`proxyAnnotationSync` is a strange name in this context. The convention in our code for mocking a `FooBar` class is to call the instance mock `fakeFooBar` and the constructor/class mock `FakeFooBar`.

Reply:
Implemented in commit number 5